src/cdk/clipboard/clipboard.md

```diff
@@ -51,7 +51,7 @@ class HeroProfile {
         pending.destroy();
       }
     };
-    setTimeout(attempt);
+    attempt();
```

Comment:
I'm guessing that the other timeout about may log the same warning on Firefox, but I couldn't manage to get something to fail after the first attempt. I didn't remove it here, because I don't have all the context on why it's wrapped in a `setTimeout` to begin with.